lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/launch/JavaLaunchCommandBuilder.java

```diff
@@ -37,19 +37,22 @@
  * Build the launch command for Java REEF processes.
  */
 public final class JavaLaunchCommandBuilder implements LaunchCommandBuilder {
+
   private static final Logger LOG = Logger.getLogger(JavaLaunchCommandBuilder.class.getName());
 
-  private static final String DEFAULT_JAVA_PATH = System.getenv("JAVA_HOME") + "/bin/" + "java";
+  private static final String DEFAULT_JAVA_PATH = "java"; // Assume java binary is in the PATH
```

Comment:
@markusweimer Yep. I think we should use `{{JAVA_PATH}}` notation in REEF job submission and implement that variable expansion in the local runtime, too. I will submit a new update for it shortly.